action.yml

```diff
@@ -53,5 +53,5 @@ outputs:
   static_web_app_url:
     description: "Url of the application"
 runs:
-  using: "docker"
```

Comment:
do we still need `using: "docker"`? i see that in our latest branch

Reply:
Yes, good call!